_posts/2018-12-27-dropping-puppet4.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Dropping Puppet 4 support
+date: 2018-12-27
+github_username: bastelfreak
+---
+
+Puppet 4 will be end of life by the December 31 2018. There isn't an official
+support matrix for the FOSS components. For Puppet enterprise this can be found
```

Comment:
sorry, the page used to have a statement about this but i think it got lost in a rewrite. I've added this block now, which hopefully clarifies the situation:

> Puppet Inc's open-source projects which are upstream of Puppet Enterprise (such as Puppet and Facter) track the maintenance and EOL dates for the PE release streams which depend on them. For example, the PE2018.1 LTS uses Puppet Agent 5.5, so the open source Puppet Agent 5.5 will receive bug fix and security updates until May 2020. For a full list of upstream components in PE releases, see the "Component Versions in Recent PE Releases" page on the docs site.